### PR TITLE
Add link back to Jupyter.org to docs

### PR DESCRIPTION
**Pull request by davidrthorn**
As per #3769, added link back to Jupyter.org to docs page.

The link is an SVG of the jupyter logo placed above the navigation breadcrumbs.

This is achieved by extending the `breadcrumbs` block in `breadcrumbs.html`, which is part of the Sphinx RTD Theme. The extension is placed in `docs/source/_templates/`.

The chosen placement of the logo has the following benefits:
1. It requires only minimal local extension of the RTD theme.
2. It works with the page's responsive design.
3. It makes semantic sense, as it sits above the navigation for the JupyterLab docs, which are  a sub-category of the Jupyter Docs.

### CODE REVIEW

**Comment by davidrthorn** [comment]
Not sure why the build checks are failing since my pull request pertains to documentation only (?)

**Comment by ellisonbg** [comment]
Could we get a screenshot of what this looks like?

**Comment by davidrthorn** [comment]
Sure. I can't put one up until Monday though...

**Comment by davidrthorn** [comment]
image

The image is quite big here, but could be made smaller.

**Comment by tgeorgeux** [comment]
@davidrthorn how difficult is this solution versus the other option you put forward:
https://user-images.githubusercontent.com/9431760/35809225-ecd469c0-0a7f-11e8-8652-318db595367d.png
I'm concerned that the png isn't explicit, and people will click it expecting to return to the home page of the documentation.

**Comment by davidrthorn** [comment]
@tgeorgeux the other solution isn't too difficult. The only cost is that it involves a bit of ugly css hacking of the template. But most modifications of this Sphinx template require some degree of that anyway.

I can give it a go over the weekend and put in a pull request perhaps?

**Comment by tgeorgeux** [comment]
@davidrthorn Let me know if you need input on implementing the other way, in the mean time I'm going to merge the simple change.

File: docs/source/_templates/breadcrumbs.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% extends '!breadcrumbs.html' %}
+
+{% block breadcrumbs %}
+  <div>
+    <a href="https://jupyter.org/documentation">
+      <img src="./_static/jupyter_logo.svg" width="120px">
```
Comment (jzf2101) [inline docs/source/_templates/breadcrumbs.html]: This is buggy for not the homepage

**Comment by tgeorgeux** [comment]
I prefer the top design, I offered the one beneath in case the Icon was going to be a problem. I don't mind showing just 'Jupyter' to be consistent with just showing 'Github' on the right but I think if we're going to ditch the 'Return to' we should also ditch the 'Edit on' from the github link.

**Pull request merged**